lib/kubeclient/http_exception.rb

```diff
@@ -1,4 +1,5 @@
-# Kubernetes HTTP Exceptions
+# TODO: remove this on next major version bump
+# Deprected http exception
```

Comment:
I guess we could use [DeprecatedConstantProxy](http://api.rubyonrails.org/classes/ActiveSupport/Deprecation/DeprecatedConstantProxy.html), but I'm not sure that library authors would be OK with bringing ActiveSupport as a dependency.